config/environments/development.rb

```diff
@@ -97,3 +97,9 @@
   end
 
 end
+
+
+ActiveMerchant::Billing::Base.mode = :test
+
+  ::GATEWAY = ActiveMerchant::Billing::StripeGateway.new(
+    :login => ENV['SECRET_KEY'])
```

Comment:
@hennevogel I'm confused, is the problem that we're not using secrets.yml here, or the name of the env variable?